reloj_checador_team03/app/assets/stylesheets/attendances.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Attendances controller here.
```

Comment:
If it's not necessary, there is no need on adding it in the project

Try to only add what is necessary in order to complete the feature you are describing in you commit message.

I think is okay for now. I mean, this is a toy-project
But take it into consideration for future contributions 😄